src/javascripts/components/search.js

```diff
@@ -13,6 +15,20 @@ var documentStore = null
 var statusMessage = null
```

Comment:
I want to do a refactor to make these instance variables on the Search class but I will do this in another PR later so this is easier to review.